### PR TITLE
chore: [TRST-1376] Create separate renovate configuration file

### PR DESCRIPTION
**Pull request by timurhamitov**
Renovate deprecated configs from npm. Created separate configuration file which we can link from the repos.

### CODE REVIEW

File: packages/renovate-config/renovate-config.json
Comment (doniyor2109) [inline packages/renovate-config/renovate-config.json:1]: @timurhamitov Maybe we should name it more specific?

Comment (timurhamitov) [reply packages/renovate-config/renovate-config.json:1]: @doniyor2109 Couldn't find a better name. please help:)

Comment (doniyor2109) [reply packages/renovate-config/renovate-config.json:1]: @timurhamitov What is the purpose of this? Maybe this helps us to choose the name

Comment (timurhamitov) [reply packages/renovate-config/renovate-config.json:1]: @doniyor2109 It's just the same configuration from `package.json` that we have. But renovate deprecated this approach. I'm trying to prepare for renovate to remove npm support at all.

Comment (timurhamitov) [reply packages/renovate-config/renovate-config.json:1]: @doniyor2109 We can call it `default.json` or `superdispatch-renovate.json` or somethin like that. But when we will extend from this config in other repos we will [provide the full path](https://docs.renovatebot.com/config-presets/#github). So we need to be aware of this.

Comment (doniyor2109) [reply packages/renovate-config/renovate-config.json:1]: Lets call it default then

Comment (timurhamitov) [reply packages/renovate-config/renovate-config.json:1]: @doniyor2109 renamed

File: packages/renovate-config/README.md [hunk 1]
```diff
@@ -11,3 +11,7 @@ Add to `renovate.json`:
   "extends": ["@superdispatch"]
 }
 ```
+
+#### Contributing
+
+Make sure to sync changes in both `renovate-config.json` and `package.json` files.
```
Comment (doniyor2109) [inline packages/renovate-config/README.md:17]:
```suggestion
Make sure to sync changes in both `default.json` and `package.json` files.
```

**Review by doniyor2109 — APPROVED** [review]

File: packages/renovate-config/README.md (hunk 1 above)
Comment (timurhamitov) [reply packages/renovate-config/README.md]: @doniyor2109 Rushed changes and missed. Thank you for noticing!

**Pull request merged**